SnapchatNavigation/Extensions/UIScrollView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UIScrollView.swift
+//  SnapchatNavigation
+//
+//  Created by Artur Chabera on 05/03/2019.
+//  Copyright © 2019 Artur Chabera. All rights reserved.
+//
+
+import UIKit
+
+extension UIScrollView {
+    static func makeHorizontalFlow(with horizontalControllers: [UIViewController], in parent: UIViewController) -> UIScrollView {
```

Comment:
I think method signature should be makeHorizontal(with controllers)
word Flow a little bit ambiguous here.